src/components/toggle-button/test/e2e/toggle-button.spec.js

```diff
@@ -0,0 +1,174 @@
+import { assert, getDriver, config } from '../../../../utils/test';
+import { VlToggleButton } from './toggle-button';
+
+const { sbUrl } = config;
+const defaultUrl = `${sbUrl}?id=custom-elements-vl-toggle-button--default`;
```

Comment:
Ik zou precies in uw testen gewoon de defaultUrl gebruiken en aanvullen, ipv een hoop constants te definiëren met 90% dezelfde string. Zoals bv https://github.com/milieuinfo/uig-webcomponents/blob/feature/cookie-consent-new/src/components/pill/test/e2e/pill.spec.js